cloud-init/daily_deb.sh

```diff
@@ -104,39 +56,28 @@ main() {
     [ $# -eq 1 ] || { bad_Usage "ERROR: Must provide a release!"; return; }
     release="$1"
 
-    TEMP_D=$(mktemp -d "${TMPDIR:-/tmp}/${0##*/}.XXXXXX") ||
-        fail "failed to make tempdir"
-    trap cleanup EXIT
-
     # program starts here
     local uuid="" name=""
     uuid=$(uuidgen -t) || { error "no uuidgen"; return 1; }
     name="cloud-init-ubuntu-daily-ppa-${uuid%%-*}"
 
-    start_container "ubuntu-daily:$release" "$name"
-
-    debug 1 "adding cloud-init daily PPA"
-    inside "$name" add-apt-repository --yes ppa:cloud-init-dev/daily ||
-        fail "failed: adding cloud-init daily PPA"
-
-    inside "$name" apt-get update ||
-        fail "failed: apt-get update"
-
-    debug 1 "downloading from PPA"
-    inside "$name" apt-get download cloud-init ||
-        fail "failed: apt-get download cloud-init"
-
-    file=$(inside "$name" ls /root/) ||
-        fail "failed: ls"
-
-    debug 1 "pulling cloud-init deb"
-    lxc file pull "$name"/root/"$file" . ||
-        fail "failed: pull file"
-
-    echo "Download successful!"
-
+    release=5ee1af40be67
```

Comment:
I assume you were testing with a working cosmic image, but should this be hardcoded?

Reply:
yeah, and removed.